sound/soc/sof/intel/hda-trace.c

```diff
@@ -51,6 +51,17 @@ int hda_dsp_trace_init(struct snd_sof_dev *sdev, u32 *stream_tag)
 		return -ENODEV;
 	}
 
+	/*
+	 * There is a HW limitation that requires the host and link DMA engines
```

Comment:
nope. the hw recommended programming sequence is to couple the host and link DMA so set the format, as done in a previous commit. This change has nothing to do with reset.